notebooks/text_models/solutions/load_text.ipynb

```diff
@@ -0,0 +1,1935 @@
+{
```

Comment:
<p>What about a more descriptive title? For example:</p><p>"Loading and pre-processing text with Keras"</p><p>"Building input pipelines with Keras for NLP"</p><p>"Building NLP input pipelines with Keras and Tensorflow"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/257/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Thank you! Updated to "Building NLP input pipelines with Keras and Tensorflow"</p>